### PR TITLE
Refactored mapping definition reader 

### PR DESCRIPTION
**Pull request by jakobbraun**
Most files only contain renamed variables or package changes.
New test: `ColumnMappingReaderTest`

### CODE REVIEW

File: src/main/java/com/exasol/adapter/dynamodb/mapping/ToJsonPropertyToColumnMapping.java [hunk 1]
```diff
@@ -11,8 +11,8 @@
  * Maps a property of a DynamoDB table and all its descendants to a JSON string.
  */
 public final class ToJsonPropertyToColumnMapping extends AbstractPropertyToColumnMapping {
-    private static final long serialVersionUID = -5414384284778565811L;
-    private final int maxLength;
+    private static final long serialVersionUID = -836984641709778657L;
+    private final int exasolStringSize;
```
Comment (AnastasiiaSergienko) [inline src/main/java/com/exasol/adapter/dynamodb/mapping/ToJsonPropertyToColumnMapping.java:15]: What does exasol string mean? Maybe it's better to rename this variable somehow

Comment (AnastasiiaSergienko) [reply src/main/java/com/exasol/adapter/dynamodb/mapping/ToJsonPropertyToColumnMapping.java:15]: Does it mean varchar?

File: src/test/java/com/exasol/adapter/dynamodb/mapping/reader/ColumnMappingReaderTest.java [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+                .add("destName", "test")//
+                .add("required", false).build();
+        assertToStringDefinitionDefaultValues(definition);
+    }
+
+    void assertToStringDefinitionDefaultValues(final JsonObject definition) {
+        final ToStringPropertyToColumnMapping columnMapping = (ToStringPropertyToColumnMapping) ColumnMappingReader
+                .getInstance()
+                .readColumnMapping("toStringMapping", definition, new DocumentPathExpression.Builder(), "test", false);
+        assertAll(// )
```
Comment (AnastasiiaSergienko) [inline src/test/java/com/exasol/adapter/dynamodb/mapping/reader/ColumnMappingReaderTest.java:44]: Do you need `)` here?

Comment (jakobbraun) [reply src/test/java/com/exasol/adapter/dynamodb/mapping/reader/ColumnMappingReaderTest.java:44]: it's closed in line 51

File: src/main/java/com/exasol/adapter/dynamodb/mapping/ToJsonPropertyToColumnMapping.java (hunk 1 above)
Comment (jakobbraun) [reply src/main/java/com/exasol/adapter/dynamodb/mapping/ToJsonPropertyToColumnMapping.java]: maybe maxSize is clearer?

File: src/test/java/com/exasol/adapter/dynamodb/mapping/reader/ColumnMappingReaderTest.java (hunk 2 above)
Comment (AnastasiiaSergienko) [reply src/test/java/com/exasol/adapter/dynamodb/mapping/reader/ColumnMappingReaderTest.java:44]: I mean you have a `)` in the comment. Looks like a typo

File: src/main/java/com/exasol/adapter/dynamodb/mapping/ToJsonPropertyToColumnMapping.java [hunk 3]
```diff
@@ -21,29 +21,29 @@
      * @param exasolColumnName     Name of the Exasol column
      * @param pathToSourceProperty {@link DocumentPathExpression} path to the property to extract
      * @param lookupFailBehaviour  {@link LookupFailBehaviour} behaviour for the case, that the defined path does not
-     * @param maxLength            Maximum string size of the Exasol column
+     * @param varcharColumnSize    Maximum string size of the Exasol column
```
Comment (AnastasiiaSergienko) [inline src/main/java/com/exasol/adapter/dynamodb/mapping/ToJsonPropertyToColumnMapping.java:24]: Change the comment too please

File: src/main/java/com/exasol/adapter/dynamodb/mapping/ToJsonPropertyToColumnMapping.java [hunk 4]
```diff
@@ -21,29 +21,29 @@
...
     }
 
     /**
      * Get the maximum string length of the Exasol column
      *
      * @return maximum string length
      */
-    public int getMaxLength() {
-        return this.maxLength;
+    public int getVarcharColumnSize() {
```
Comment (AnastasiiaSergienko) [inline src/main/java/com/exasol/adapter/dynamodb/mapping/ToJsonPropertyToColumnMapping.java:45]: And here

File: src/main/java/com/exasol/adapter/dynamodb/mapping/ToStringPropertyToColumnMapping.java [hunk 5]
```diff
@@ -38,12 +38,12 @@ public ToStringPropertyToColumnMapping(final String exasolColumnName,
      * 
      * @return maximum size of Exasol VARCHAR
```
Comment (AnastasiiaSergienko) [inline src/main/java/com/exasol/adapter/dynamodb/mapping/ToStringPropertyToColumnMapping.java:39]: Here too

**Pull request merged**